### PR TITLE
fc-app-168: Don't truncate MC inputs upload JSON [risk: low]

### PR DESCRIPTION
**Pull request by jmthibault79**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [ ] **Product Owner** sign off
- [x] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by jmthibault79** [comment]
New `:truncate-preview?` prop for import_data/Page
- was truncating actual MC inputs upload.  fc-app #168

Looking at #1327 may be helpful for context

File: src/cljs/main/broadfcui/page/workspace/data/tab.cljs [hunk 1]
```diff
@@ -48,7 +48,8 @@
            (case last-crumb-id
              :file-import
              [import-data/Page
-              (select-keys props [:workspace-id :import-type :on-data-imported])]
+              (merge (select-keys props [:workspace-id :import-type :on-data-imported])
+                     {:truncate-preview? true})]
```
Comment (jmthibault79) [inline src/cljs/main/broadfcui/page/workspace/data/tab.cljs:52]: `import-data/Page` is used for preview only here, so truncating is fine.

File: src/cljs/main/broadfcui/page/workspace/summary/attribute_editor.cljs [hunk 2]
```diff
@@ -123,7 +123,8 @@
               :content [:div {:style {:width 720 :backgroundColor "white" :padding "1em"}}
                         [import-data/Page (merge (select-keys props [:workspace-id])
                                                  {:on-data-imported (:request-refresh props)}
-                                                 {:import-type "workspace-attributes"})]]}])
+                                                 {:import-type "workspace-attributes"}
+                                                 {:truncate-preview? true})]]}])
```
Comment (jmthibault79) [inline src/cljs/main/broadfcui/page/workspace/summary/attribute_editor.cljs:127]: `import-data/Page` is used for preview only here, so truncating is fine.

**Review by gdlex4015 — APPROVED** [review]

**Review by davidangb — APPROVED** [review]
one tiny nit: since you changed the `unmatchedVariables` to be > 4k, also add a comment somewhere in the `Upload inputs.json` test definition, since those are pretty far apart in code. Otherwise, did not test but looks 👍

**Pull request merged**